packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/header-mixed.twig

```diff
@@ -69,14 +69,11 @@
     {% endif %}
   </div>
 
-  <nav class="ma__header__nav" aria-label="main navigation" id="main-navigation" role="navigation">
+  <nav class="ma__header__nav" aria-label="Main navigation" id="main-navigation" role="navigation">

```

Comment:
Should the the arial-label on `<nav>` be consistent with the mobile version (https://github.com/massgov/mayflower/pull/1401/files#diff-2cf6b391741fd8a18285aade0abe3524d313ec9ebc247bb96ec4c83e7ca08682R12)?

Reply:
L.12 is the blue bar at the top of the page.  That includes the mobile component.  It contains the utility nav for desktop, the mobile component (menu button, the logo, the search, the main nav and the utility nav).  So, I put aria-label as "Main **menu**" to differentiate from the main nav component (topic links) itself.

L. 72 is the horizontal nav.  So, it only contains the main nav component (topic links) but nothing else, so its aria-label is set as "Main **navigation**"

I think both are labeled as "Main navigation" would be more confusing in desktop display.

<img width="1267" alt="home-desktop" src="https://user-images.githubusercontent.com/9633303/116429624-0c0e2d00-a814-11eb-95a5-0cc69747f88c.png">

Reply:
Ok I think I understand where my confusion came from -- due to the HTML structure differences between desktop and mobile, the navigation role is assigned to two different things:
1. On desktop - nav contains only the mainNav - with the ariaLabel
2. On mobile - nav contains both the mainNav and utilityNav

Reply:
desktop: ![Screen Shot 2021-04-28 at 1 30 08 PM](https://user-images.githubusercontent.com/5789411/116449109-cf4c3100-a827-11eb-9345-a79f5df9b41e.png)

mobile: ![Screen Shot 2021-04-28 at 1 29 37 PM](https://user-images.githubusercontent.com/5789411/116449062-c3f90580-a827-11eb-8bef-6bae4134fa0d.png)

Reply:
I wonder if it makes sense to remove the nav role from "Main menu" and assign that on mobile to be on the "_main-nav" div or the "ul" element, in order to make the roles and labels consistent between mobile and desktop